mgplvm/kernels/linear.py

```diff
@@ -0,0 +1,113 @@
+import torch
+from torch import nn, Tensor
+from .kernel import Kernel
+from typing import Tuple, List
+import numpy as np
+
+
+class Linear(Kernel):
+    name = "Linear"
+
+    def __init__(self, n: int, d: int, scale=None, learn_scale=False, Y=None):
+        '''
+        n is number of neurons/readouts
+        distance is the distance function used
+        d is the dimensionality of the group parameterization
+        scaling determines wheter an output scale parameter is learned for each neuron
+        
+        learn_scale : learn an output scaling parameter (similar to the RBF signal variance)
+        '''
+        super().__init__()
+
+        if scale is not None:
+            _scale = torch.tensor(scale).sqrt()
```

Comment:
I think we should consistently provide  'scale' as the sqrt version so it has units of activity instead of squared activity. I think we also provide the squared version (i.e. noise variance) as input in the Gaussian likelihood which I would suggest changing to the std.

Reply:
I think I prefer that the scale parameter has units of variance instead of activity. Previously, we made sure that `alpha` is positive by parameterising it with a `softplus` function, then squaring it again to get the variance. It just seems much easier to deal directly work with the variance `scale`, where the raw parameter `_scale` can have any sign.

Reply:
In the latest commit, I've made `scale` have units of activity. I define a new variable `scale_sqr` internally which scales the parameter `_scale`, which can be either positive or negative. To avoid confusion I've also added a property `scale` which is just `_scale.abs()`. I'm reasonably happy this, thoughts?

Reply:
Yeah scale should be the std, that is the convention in scipy distributions and others I have seen. Looks fine now

Reply:
yea fair, I think I've changed all of them in the tests as well, but it might also be worth double checking.